src/views/ForgotPasswordPageChange/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import { Link, Route } from "react-router-dom";
+import LoginPage from "../LoginPage";
+
+var requirements = {
```

Comment:
We actually don't need a requirements object at all so you can get rid of it entirely